portal-api/src/v1/controllers/companies.controller.js

```diff
@@ -0,0 +1,13 @@
+const externalApi = require('../../external-api/api');
+
+const getCompanyByRegistrationNumber = async (req, res) => {
```

Comment:
Nice job here keeping to the `router -> controller -> external-api` pattern. I think this is a great idea, in case we need to add any additional logic here or reuse the `external-api` `companies` service elsewhere. Very nice.